scripts/build-win.py

```diff
@@ -83,6 +82,9 @@ def gngen(arch, sio_root, ffmpeg_root, ssl_root, msdk_root, quic_root, scheme, t
         gn_args.append('owt_include_tests=false')
     if cloud_gaming:
         gn_args.append('owt_cloud_gaming=true')
+        gn_args.append('enable_libaom=false')
+    else
```

Comment:
That's wrong syntax. You are missing `:`:
```
  File ".\\scripts\\build-win.py", line 86
    else
       ^
SyntaxError: invalid syntax
```

Reply:
Fixed. Thanks for pointing out.

Reply:
I verified fix - it works for me on Windows.